saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -527,6 +527,68 @@ def test_checkout_create_check_lines_quantity_multiple_warehouse(
     assert data["errors"][0]["field"] == "quantity"
 
 
+def test_checkout_create_sets_country_from_shipping_address_country(
```

Comment:
Maybe we should set `DEFAULT_COUNTRY`  for any other country than `US`?

Reply:
Yes but we want to test scenario where your selected country is other than one from shipping address. `DEFAULT_COUNTRY` id the one I will get from `request.user.country`?

Reply:
Done